### PR TITLE
LPS-76780 W.I.P. on portlet3

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

**Comment by julien** [comment]
ci:test

File: modules/apps/foundation/frontend-js/frontend-js-web/bnd.bnd [hunk 1]
```diff
@@ -151,6 +151,7 @@ Liferay-Top-Head-JS-Resources:\
 	/liferay/portal.js,\
 	/liferay/portlet.js,\
 	/liferay/workflow.js,\
+	/liferay/portal.bundle.js,\
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-js/frontend-js-web/bnd.bnd]: Where is this coming from?

File: modules/apps/foundation/frontend-js/frontend-js-web/package.json [hunk 2]
```diff
@@ -43,6 +43,14 @@
 		"webpack-merge": "^4.1.1"
 	},
 	"jest": {
+		"collectCoverage": true,
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-js/frontend-js-web/package.json]: We should not commit this to the repo, even if useful during development, as CI has no way to handle this, and I'm sure people will end up comiting the coverage output by mistake.

Comment (julien) [reply modules/apps/foundation/frontend-js/frontend-js-web/package.json]: Ok, I'll remove that. Thanks

File: modules/apps/foundation/frontend-js/frontend-js-web/package.json [hunk 3]
```diff
@@ -59,6 +67,8 @@
 		"checkFormat": "npm run csf",
 		"csf": "csf src/**/liferay/**/*.js test/**/*.js",
 		"format": "npm run csf -- -i",
+		"lint": "eslint src/**/*.es.js test/**/*.es.js",
+		"prettier": "prettier-eslint src/**/*.es.js test/**/*.es.js",
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-js/frontend-js-web/package.json]: This should not be here. `format` is the `prettier` equivalent and `checkFormat` the `lint` one. Those are the tasks we should be running and no other.

File: modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portal.es.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import register from './portlet_hub/register.es';
+
+export {register};
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portal.es.js]: Isn't this already `global.es.js`? Why duplicate it?

File: modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portlet_hub/MockData.es.js [hunk 5]
```diff
@@ -0,0 +1,184 @@
+/**
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portlet_hub/MockData.es.js]: Is this file for testing? Doesn't feel like this should be in sources...

Comment (julien) [reply modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portlet_hub/MockData.es.js]: Ok I'll move it to tests.

File: modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portal.es.js (hunk 4 above)
Comment (julien) [reply modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portal.es.js]: @jbalsas there are changes that I got from https://github.com/jbalsas/liferay-portal/pull/1009/files - I had no idea if they were needed or not.

Comment (jbalsas) [reply modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portal.es.js]: This file should be removed, same as the entry in the bnd bnd file. The entry point should be `globals.es.js` as it is already in master.

**Pull request closed without merging**